third_party/NordicSemiconductor/drivers/radio/platform/clock/nrf_802154_clock_sdk.c

```diff
@@ -65,12 +65,12 @@ static void clock_handler(nrf_drv_clock_evt_type_t event)
 
 void nrf_802154_clock_init(void)
 {
-    // Intentionally empty.
+    nrf_drv_clock_init();
```

Comment:
It's a little suspicious for me, why this has been changed. We used empty functions for each previous radio driver release.

Reply:
Same for `nrf_802154_clock_deinit` function.